### PR TITLE
Story3

### PR DESCRIPTION
**Pull request by jennymegan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,14 +9,35 @@
     <title>Jarver's Crypt Maze</title>
 </head>
 <body>
+
+    <!-- The Win Modal -->
```
Comment (mporam) [inline index.html]: these comments feel redundant. the `id` litterally says the same thing

File: index.html [hunk 2]
```diff
@@ -35,40 +56,38 @@ <h2 class="instructionsTitle">Instructions</h2>
...
-            <img src="images/door.png" alt="finish door" class="win">
+            <img src="images/door.png" alt="finish door" class="winningSquare">
         </div>
-
-<!--        images-->
-        <div class="ghost">
-            <img src="images/ghost.png" alt="ghost" class="collision">
+        <div class="ghost collision">
+            <img class="ghostImage" src="images/ghost.png" alt="ghost">
+            <div class="ghostHitbox collision"></div>
```
Comment (mporam) [inline index.html]: why the random empty `div`?

File: script.js [hunk 3]
```diff
@@ -1,9 +1,120 @@
 
+// Get the modal
+let winModal = document.querySelector("#gameWinPopup")
+let loseModal = document.querySelector("#gameLosePopup")
+
+// code here to activate win modal displaying
+    let win = function() {
```
Comment (mporam) [inline script.js]: why the indentation? and why the use of an anonymous function?

File: script.js [hunk 4]
```diff
@@ -1,9 +1,120 @@
...
+    }
+
+// code here to activate lose modal displaying
+
+let lose = function() {
+   loseModal.style.display = "block"
+}
+
+//close the modal because there is no restart button
+document.querySelector('#closeLoseModal').addEventListener('click', () =>{
```
Comment (mporam) [inline script.js]: missing space before the `{`

File: script.js [hunk 5]
```diff
@@ -1,9 +1,120 @@
 
+// Get the modal
+let winModal = document.querySelector("#gameWinPopup")
```
Comment (mporam) [inline script.js]: do these need to be variables? unless they change, make them constants

File: script.js [hunk 6]
```diff
@@ -1,9 +1,120 @@
...
+function listenForWinning() {
+    document.querySelector('.winningSquare').addEventListener('mouseenter', (e) => {
+        win()
+        e.stopPropagation();
+    })
+}
+
+//call functions from here
+
+let ghost = document.querySelector('.ghost');
```
Comment (mporam) [inline script.js]: again, do these need to be variables?

File: script.js [hunk 7]
```diff
@@ -1,9 +1,120 @@
...
+let pumpkin = document.querySelector('.pumpkinImage');
+let reaper = document.querySelector('.grim_reaperImage');
+let lineSeven = document.querySelector('.line_seven');
+
+let start = document.querySelector('.start_area');
```
Comment (mporam) [inline script.js]: doesnt look liek you use this more than once? does it need to be in a variable at all??

File: script.js [hunk 8]
```diff
@@ -1,9 +1,120 @@
...
+    listenForWinning();
+})
+
+function startAnimation() {
+    ghost.animate([
+        {transform: 'translateY(0px)'},
+        {transform: 'translateY(150px)'},
+        {transform: 'translateY(0px)'}
+    ], {
+        // timing options
```
Comment (mporam) [inline script.js]: unnecessary

File: style.css [hunk 9]
```diff
@@ -18,26 +16,39 @@ html, body {
...
     background-color: #874000;
     font-family: "Creepster", sans-serif;
     font-size: 4rem;
     width: 800px;
-    text-align: center;
     margin: 0 auto;
     padding: 20px;
 }
 
+.gameBox {
```
Comment (mporam) [inline style.css:36]: this is basically duplicate code for the `.maze` class, feels like you could simplify it

File: style.css [hunk 10]
```diff
@@ -259,3 +279,56 @@ html, body {
     font-size: 1.5rem;
     font-family: "Fredericka the Great";
 }
+
+
+/* under here is modal */
```
Comment (mporam) [inline style.css]: really? i wouldnt have guessed that from the class name of `modal`

File: style.css [hunk 11]
```diff
@@ -259,3 +279,56 @@ html, body {
...
+.modal {
+    position: fixed;
+    display: none; /* Hidden by default */
```
Comment (mporam) [inline style.css]: these comments are getting rediculous now. `display: none` means its hidden!? no way!!

File: style.css [hunk 12]
```diff
@@ -259,3 +279,56 @@ html, body {
...
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0, 0, 0); /* Fallback color */
+    background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+    z-index: 30;
+}
+
+/* Modal Content */
```
Comment (mporam) [inline style.css]: stop it!

**Review by mporam — CHANGES_REQUESTED** [review]
please also fix the cursor on the start base coffin and the unnecessary divs everywhere

File: script.js (hunk 3 above)
Comment (davin2020) [reply script.js]: moved to relevant function

File: script.js (hunk 4 above)
Comment (davin2020) [reply script.js]: ok

File: script.js (hunk 5 above)
Comment (davin2020) [reply script.js]: ok

File: style.css (hunk 9 above)
Comment (davin2020) [reply style.css:36]: removed unnecessary code

File: style.css (hunk 10 above)
Comment (davin2020) [reply style.css]: removed

File: style.css (hunk 11 above)
Comment (davin2020) [reply style.css]: removed

File: style.css (hunk 12 above)
Comment (davin2020) [reply style.css]: removed

File: index.html (hunk 1 above)
Comment (davin2020) [reply index.html]: removed

File: index.html (hunk 2 above)
Comment (davin2020) [reply index.html]: removed

File: script.js (hunk 8 above)
Comment (davin2020) [reply script.js]: ok

File: index.html [hunk 13]
```diff
@@ -52,14 +45,13 @@ <h2 class="instructionsTitle">Instructions</h2>
             </div>
         </div>
     </div>
-  
-  <!-- Game screen -->
+
   <div id="gameScreen">
     <h1>Jarver's Crypt</h1>
-    <div class="gameBox">
+    <div class="gameBox ">
```
Comment (mporam) [inline index.html]: why?

File: index.html [hunk 14]
```diff
@@ -52,14 +45,13 @@ <h2 class="instructionsTitle">Instructions</h2>
...
         <div class="start_area">
             <h2>Start</h2>
-            <img src="images/coffin.png" alt="a coffin">
+            <img src="images/coffin.png" alt="a coffin" >
```
Comment (mporam) [inline index.html]: why?

File: script.js [hunk 15]
```diff
@@ -44,22 +33,23 @@ function listenForCollisions() {
...
         e.stopPropagation();
     })
 }
 
 //call functions from here
 
 let ghost = document.querySelector('.ghost');
-let pumpkin = document.querySelector('.pumpkinImage');
-let reaper = document.querySelector('.grim_reaperImage');
+let pumpkin = document.querySelector('.pumpkin');
```
Comment (mporam) [inline script.js]: do these need to change? if not, make them constants

File: script.js (hunk 7 above)
Comment (mporam) [reply script.js]: this seems like it can be deleted??

**Review by mporam — CHANGES_REQUESTED** [review]

File: script.js (hunk 7 above)
Comment (davin2020) [reply script.js]: const start will be used in later stories

File: script.js (hunk 6 above)
Comment (davin2020) [reply script.js]: changed to consts

File: index.html (hunk 13 above)
Comment (davin2020) [reply index.html]: ok

File: index.html (hunk 14 above)
Comment (davin2020) [reply index.html]: ok

File: script.js (hunk 15 above)
Comment (davin2020) [reply script.js]: made consts

**Review by mporam — APPROVED** [review]

**Pull request merged**